src/App.css

```diff
@@ -1,3 +1,260 @@
-*body {
-  background-color: white;
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  background-color: #f5f6fa;
+}
+
+#root {
+  margin: 5% 4%;
+  display: flex;
+  background-color: #fafafa;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+}
+
+.app {
+  width: 100%;
+}
+
+.app nav {
+  width: 100%;
+  background: white;
+  border-bottom: 1px solid #dfdfdf;
+  height: 85px;
+  margin-top: -14px;
+}
+
+.app ul {
+  display: flex;
+  gap: 32px;
+  list-style-type: none;
+  align-items: center;
+  padding: 0 2%;
+}
+
+.img {
+  display: flex;
+  justify-content: flex-end;
+  width: 64%;
+}
+
+.img img {
+  border: 1px solid #e8e8e8;
+  border-radius: 34px;
+  cursor: pointer;
+  padding: 8px;
+  width: 44px;
+}
+
+.headerName {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+  padding-left: 5%;
+}
+
+.header-ul a {
+  opacity: 1;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  color: #121212;
+  letter-spacing: 1.9px;
+  text-decoration: none;
+}
+
+hr {
+  height: 0.125rem;
+  border: 1px solid #e8e8e8;
+  width: 100%;
+}
+
+form {
+  display: flex;
+  padding: 0 2%;
+  justify-content: flex-start;
+  gap: 17px;
+}
+
+.form-div {
+  padding: 30px;
+}
+
+.form-div h2 {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+#title-input {
+  padding: 12px;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  font-size: 1rem;
+  letter-spacing: -0.15px;
+  color: #121212;
+  width: 450px;
+}
+
```

Comment:
- [ ] Please make sure that the form elements are expanded to take up all of the available horizontal space, just like in the provided Zeplin design.
- [ ] Also please align the `Add new book` heading right above the title input field as currently your heading is slightly away from it. Please see the following screenshots for reference. 
 
Your Implementation
---
<img width="1397" alt="Screenshot 2023-07-29 at 9 55 49 AM" src="https://github.com/kerlos-bahaa/Bookstore-react/assets/44798044/5cdd821e-ea4a-41ff-8e13-49f4fce562bd">

Provided Zeplin design
------
<img width="988" alt="Screenshot 2023-07-29 at 9 55 35 AM" src="https://github.com/kerlos-bahaa/Bookstore-react/assets/44798044/6ccc4561-4ad7-48cc-aadb-253885149685">